config/defaults.go

```diff
@@ -0,0 +1,308 @@
+package config
+
+import (
+	"github.com/spf13/viper"
+)
+
+// DefaultConfigFilePath is in the root folder by default
+const DefaultConfigFilePath = "config.json"
+
+// General defaults
+const (
+	DebugDefault              = true
+	DebugProfilingDefault     = true
+	DisableITCDefault         = true
+	ImportBlockHeadersDefault = ""
+	RequestLoggingDefault     = true
+)
+
+// Authentication defaults
+const (
+	AuthAdminKeyDefault        = "xpub661MyMwAqRbcFaYeQLxmExXvTCjw9jjBRpifkoGggkAitXNNjva4TStLJuYjjEmU4AzXRPGwoECjXo3Rgqg8zQqW6UPVfkKtsrogGBw8xz7"
+	AuthRequireSigningDefault  = false
+	AuthSchemeDefault          = "xpub"
+	AuthSigningDisabledDefault = true
+)
+
+// Beef defaults
+const (
+	UseBeefDefault                  = true
```

Comment:
I would switch UseBeefDefault to false, because if someone uses Beef he/she needs to set Pulse endpoint also. It means that the default value (true) without changing few options in this section won't work properly.